### ISSUE TITLE
Why only Docker ?

### ISSUE THREAD

**Issue by alx-xlx** [vrtmrz/obsidian-livesync#242]
I think there should be an alternative way to setup the server other than docker.
I see no articles that I could find that would allow me to get started with the server where I had the data on my own local server.

I think that would be a good addition we can host it locally lets say on a raspberry pi or something

**Comment by vrtmrz** [comment]
Thank you for your idea!
Yes, we can install CouchDB on any server directly. 
However, it can be quite diverse. Actually, [the official documentation of CouchDB Installation](https://docs.couchdb.org/en/stable/install/index.html) is so long.

Although, might be helpful to know that we have several ways. 
What do you think about adding some notes introducing the official documentation?

**Comment by alx-xlx** [comment]
You are correct infact the docker is the only way to make couchdb work on ARM64 devices. I tried to compile their source-code but gave up. Its old piece of tool and few libraries have gone missing.

So basically its couchdb issue and not obsidian-livesync. My apologies !

**Comment by alx-xlx** [comment]
However I have two questions

1. When I run the server I don't know why but the database gets deleted on the server. Only when I connect my device it gets created. Is this normal ?

2. I don't know what would go into the `database address` ?

![](https://i.imgur.com/F5gwSlh.png)
